lxd/db/network_acls.go

```diff
@@ -312,3 +314,24 @@ func (c *Cluster) DeleteNetworkACL(id int64) error {
 		return err
 	})
 }
+
+// GetNetworkACLURIs returns the URIs for the network ACLs with the given project.
+func (c *ClusterTx) GetNetworkACLURIs(projectID int) ([]string, error) {
+	stmt, err := c.prepare(`
+SELECT projects.name, networks_acls.name from networks_acls
+JOIN projects ON networks_acls.project_id = projects.id
+WHERE networks_acls.project_id = ?`)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to prepare statement for network acl: %w", err)
+	}
+
+	code := cluster.EntityTypes["network acl"]
+	formatter := cluster.EntityFormatURIs[code]
```

Comment:
@masnax not for this PR probably, but I can see us removing ` cluster.EntityFormatURIs` and `cluster.EntityURIs` in the near future, as URL generation really doesn't need to be in the DB layer.